code/modules/client/preference_setup/loadout/loadout_utility.dm

```diff
@@ -29,6 +29,11 @@
 	display_name = "the traveler's guide to vir"
 	path = /obj/item/weapon/book/codex/lore/vir
 	cost = 0
+	
+/datum/gear/utility/news
+	display_name = "daedalus political almanac"
```

Comment:
Fill me in on the Daedalus part?
Also, if it's news, rather than like, an encyclopedia, Almanac doesn't feel like the right option?

Reply:
Daedalus being Icarus's father. The Daedalus Dispatch is one of exactly 2 named newsnets other than the newscaster ones and I figured it'd be good to be upfront about the fact that the articles are probably gonna skew to the technocon.

Gimme a sec to change names.